richards-benchmark/README.md

```diff
@@ -0,0 +1,15 @@
+# Richards benchmark
+
+Richards is an OS task scheduler simulation benchmark, designed for 
+comparing system implementation languages. The benchmark was originally
+implemented in BCPL by Martin Richards.
+See https://www.cl.cam.ac.uk/~mr10/Bench.html for details.
+This is a CHERI-compatible version of the Richards benchmark, with
+optional support for BoehmGC. The code is derived from Stefan Marr's
+classic benchmarks repository at
+https://github.com/smarr/Classic-Benchmarks
+
+The Richards benchmark does not have a licence. Instead, Martin Richards
```

Comment:
I checked this with Martin in 2012 and he said:

> Yes, everything in my bench distribution is in the public domain except for the directory sunjava.

so it does have a licence (which is good!).

Reply:
good ... so do we need a LICENSE.txt ? What should it say?

Reply:
Good idea! We can just say "Placed into the public domain by Martin Richards."